tools/codegen/schema/schema_file.go

```diff
@@ -9,14 +9,18 @@ import (
 
 func GenerateGoCode(input genconfigmapper.Resource) string {
 	schemaAttrs := GenerateSchemaAttributes(input.Schema.Attributes)
+	models := GenerateTypedModels(input.Schema.Attributes)
 
 	tmplInputs := codetemplate.SchemaFileInputs{
 		PackageName:      input.Name,
-		Imports:          schemaAttrs.Imports,
-		SchemaAttributes: schemaAttrs.Result,
+		Imports:          append(schemaAttrs.Imports, models.Imports...),
+		SchemaAttributes: schemaAttrs.Code,
+		Models:           models.Code,
 	}
 	result := codetemplate.ApplySchemaFileTemplate(tmplInputs)
 
+	print(result.String())
```

Comment:
Maybe add some header before printing?

Reply:
thanks, left unintentionally, prefer to remove as it could be too verbose when running the make command